app/views/instances/_show_waiting.erb

```diff
@@ -6,14 +6,14 @@
         <h5>Players:</h5>
         <p> <%= players.count %> / <%= instance.max_players %></p> <!--number of players here -->
       </div>
-        <% players.each do |player| %>
-          <% user = User.find(player.user_id)%>
-          <div class>
-            <div id="card" class="card shadow-sm p-3 my-1 rounded">
-              <%= user.nickname%>
-            </div>
+      <% players.each do |player| %>
```

Comment:
In case anyone wonder - I just corrected the indentation here, I am a bit paranoid with divs now 😅